src/backend/executor/bridge.ts

```diff
@@ -57,6 +57,12 @@ export class ExecutorBridge implements Disposable {
 
         workspace.onDidSaveTextDocument(this.analyzeOnSave, this, ctx.subscriptions);
         workspace.onDidChangeConfiguration(this.updateDatabasePaths, this, ctx.subscriptions);
+        workspace.onDidChangeConfiguration((e) => {
+            // Check the version only if the CodeChecker executable path is changed.
+            if (!e.affectsConfiguration('codechecker.executor.executablePath')) { return; }
```

Comment:
`affectsConfiguration` accepts a section, not a full setting ID, so only `codechecker.executor` here.